client/packages/invoices/src/Returns/modals/SupplierReturn/ItemSelector.tsx

```diff
@@ -40,13 +40,11 @@ export const ItemSelector: FC<ItemSelectorProps> = ({
             disabled={disabled}
             currentItemId={itemId}
             onChange={newItem => newItem && onChangeItemId(newItem.id)}
-            includeNonVisibleWithStockOnHand
+            filter={{ hasStockOnHand: true }}
             extraFilter={
               disabled
                 ? undefined
-                : item =>
-                    item.availableStockOnHand !== 0 &&
-                    !existingItemIds?.some(id => id === item.id)
+                : item => !existingItemIds?.some(id => id === item.id)
```

Comment:
We have a lot of uses of StockItemSearchInput that have an extra filter like this... it might bite us too? (Filter excludes all items that already exist in the invoice.)

E.g. I have 150 items visible, I include 130 in an order... and then, of the first 100, only 7 aren't in the order, so I show those in the item selector - that's not enough to trigger the scroll/pagination, can't access the other 23...

Haven't seen this raised as a bug, seems like an edge case, but possibly we should have an "excludeItems" array on the API filter, rather than frontend filter? Can raise an issue?